### PR TITLE
test: add e2e tests for SUBSCRIBE/UNSUBSCRIBE_ADDRESS

### PR DESCRIPTION
**Pull request by bslabiak**

### CODE REVIEW

File: test/e2e/tests/unsubscribe-address.e2e.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    const randomLovelaceValue = (Math.floor(Math.random() * (1500000 - 1000000 + 1)) + 1000000).toString();
+    const { transaction } = await buildTx(sendingMnemonic, randomLovelaceValue, receivingAddress);
+    await ws1.sendAndWait('PUSH_TRANSACTION', {
+      txData: transaction.to_hex(),
+    });
+
+    // Wait for a new block
+    await ws2.waitForSubscriptionMessages(1, 160_000);
+
+    // Ensure the alternative client got the new block while the standard one didn't
```
Comment (slowbackspace) [inline test/e2e/tests/unsubscribe-address.e2e.test.ts:55]: don't forget to update comments! :)

File: test/e2e/tests/subscribe-address.e2e.test.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      const { transaction, txHash } = await buildTx(fixture.sendingWallet?.mnemonic, randomLovelaceValue, fixture.receivingWallet?.address);
+      await ws.sendAndWait('PUSH_TRANSACTION', {
+        txData: transaction.to_hex(),
+      });
+
+      //Check subscription message
+      const [message] = await ws.waitForSubscriptionMessages(1, 150_000);
+      expect(message.data).toMatchObject(fixture.subscribe_message_schema);
+      expect(message.data[0].txHash).toMatchObject(txHash);
+      expect(message.data[1].txHash).toMatchObject(txHash);
```
Comment (slowbackspace) [inline test/e2e/tests/subscribe-address.e2e.test.ts:33]: I am actually confused why the message contains the same tx twice. What am I missing?

Comment (bslabiak) [reply test/e2e/tests/subscribe-address.e2e.test.ts:33]: I subscribed 2 addresses and did tx between them, thats why subscribtion contains same txHash.

Comment (slowbackspace) [reply test/e2e/tests/subscribe-address.e2e.test.ts:33]: ohh I completely missed that there are 2 addresses not just one 🤦‍♂️  Thanks

**Review by slowbackspace — APPROVED** [review]
splendid!

**Pull request merged**